WooCommerce/Classes/ViewRelated/Feature Highlight/FeatureHighlightStore.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+struct FeatureHighlightStore {
```

Comment:
💭 I'd suggest avoiding the suffix `Store` so that it doesn't sound like it's a subclass of Yosemite Store. The name of the struct "FeatureHighlight" also sounds a bit generic. Maybe it can be called like `ProductDescriptionAITooltipUseCase`.

Reply:
future suggestion: this struct can be unit tested

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10005/commits/dcb8162d80c18312e9e233fc8b6dd9b271a37ea3

> future suggestion: this struct can be unit tested

Will add this in a future PR.